infrastructure/hosts/catcolab-next/backend.nix

```diff
@@ -183,17 +210,28 @@ in {
     };
 
     security.sudo.extraRules = [{
-        users = [ "catcolab" ]; 
+        users = [ "catcolab" "deployuser" ];
         commands = [
             { command = "/run/current-system/sw/bin/systemctl start automerge"; options = [ "NOPASSWD" ]; } 
             { command = "/run/current-system/sw/bin/systemctl stop automerge"; options = [ "NOPASSWD" ]; } 
             { command = "/run/current-system/sw/bin/systemctl restart automerge"; options = [ "NOPASSWD" ]; }
             { command = "/run/current-system/sw/bin/systemctl start backend"; options = [ "NOPASSWD" ]; } 
             { command = "/run/current-system/sw/bin/systemctl stop backend"; options = [ "NOPASSWD" ]; } 
             { command = "/run/current-system/sw/bin/systemctl restart backend"; options = [ "NOPASSWD" ]; }
+            { command = "${updateScript}/bin/catcolab-update";  options = [ "NOPASSWD" ]; }
```

Comment:
The way the existing scripts are created and used is strange blend of manual unixy install steps and declarative nixy configuration. Mixing them seems wrong so I added the new one the nix way. I'm not sure if this was the right call